app/_types/forms.ts

```diff
@@ -13,15 +13,17 @@ export type SingleTradeRouteForm = TradeRouteFilters & {
 };
 
 export type MultiTradeRouteForm = TradeRouteFilters & {
-  startSystem: string;
+  startSystem?: { value: number };
   startStation?: string;
-  endSystem?: string;
+  finishSystem?: { value: number };
+  maxHopCount?: string;
 };
 
 export type TradeRouteFilters = {
```

Comment:
Just noticed this on this change but if this is done in other commits, we should discuss and close on the pattern of writing types. We had been following the pattern of `I{Resource}` e.g., `ICommodity`. I can connect on discord to discuss this quickly otherwise if you are good with the pattern, we can switch to that.

Reply:
@nitishdhar I've been using this for the responses, but not the form filters. The reason is probably because I tend to use interfaces to define entities, and types to define everything else (I admit I've not been 100% consistent with this, usually where I've copypasted from an existing page and have left it as is for consistency) - if you look at the station/system/trade route type files you'll see that I've used types for form definitions, and interfaces for the responses, and I've use the your naming standard for interfaces - IStation, ISystem, etc.

Am happy to update the form type names if you're keen to make this sort of naming standard across everything, but if we do then where do we draw the line with it? Should these be IFormProps and ISubmitProps?  Does every type need to have a prefix?

![image](https://github.com/ed-pilots-network/frontend/assets/319883/40919c62-3e1b-4e13-8faa-695f72d7715e)

I admit I've never really been on board with this sort of naming, it all feels very superfluous - we know it's an interface, putting an I at the start doesn't really provide any extra information, it feels like something people did in old times when their IDEs couldn't do effective syntax highlighting so they could tell what something was at a glance.

Reply:
@chrismou I get what you are saying and my intent here is more about consistency than picking one approach over the other. Honestly I am fine with either of the two as long as there is a consistent pattern across the codebase. You are also right about this being an old pattern and that’s just something I have been using but I will not die on this hill. You can resolve this for now, we can discuss / define a consistent pattern in a different issue. Thanks for clarifying things in detail 👍

Reply:
Agreed - whatever we go for, the most important thing really is consistency